forge-gui/res/cardsfolder/upcoming/tivit_seller_of_secrets.txt

```diff
@@ -0,0 +1,13 @@
+Name:Tivit, Seller of Secrets
+ManaCost:3 W U B
+Types:Legendary Creature Sphinx Rogue
+PT:6/6
+K:Flying
+K:Ward:3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigVote | TriggerDescription$ Whenever CARDNAME enters the battlefield or deals combat damage to a player, starting with you, each player votes for evidence or bribery. For each evidence vote, investigate. For each bribery vote, create a Treasure token.
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigVote | TriggerZones$ Battlefield | Secondary$ True | TriggerDescription$ Whenever CARDNAME enters the battlefield or deals combat damage to a player, starting with you, each player votes for evidence or bribery. For each evidence vote, investigate. For each bribery vote, create a Treasure token.
+SVar:TrigVote:DB$ Vote | Defined$ Player | VoteType$ Evidence,Bribery | VoteEvidence$ DBEvidence | VoteBribery$ DBBribery | EachVote$ True

```

Comment:
i need to check if the tokens are created at once or not for replacement effects

Reply:
Looks like it should be everyone votes, then Investigate X times, then make Y treasure tokens. Looks like we need to fix _Expropriate_ as well:
![image](https://user-images.githubusercontent.com/103371817/166464824-d174c871-d0e4-49a0-b215-3db0e93ccc74.png)

Reply:
Does the script need changing here?

Reply:
i don't know how to config Vote Effect right now, but in theory it should do:
```
Investigate | Num$ X
Treasure | Num$ Y
X=NumEvidence
Y=NumBribery
```

probably using `StoreVoteNum`

rules wise:
* first it does X times investigate (creating 1 clue each time, important for replacement effects)
* then it does create Y treasure at once

Reply:
OK, might be slightly beyond my current understanding but I'll give it a shot.

Reply:
checkout other `Council's dilemma` cards like `Lieutenants of the Guard` using VoteNum
```
SVar:TrigChangeZone:DB$ Vote | Defined$ Player | StoreVoteNum$ True | VoteType$ Strength,Numbers | SubAbility$ DBVoteStrength | AiLogic$ StrengthOrNumbers
SVar:DBVoteStrength:DB$ PutCounter | Defined$ Self | CounterType$ P1P1 | CounterNum$ VoteNumStrength | SubAbility$ DBVoteNumbers
SVar:DBVoteNumbers:DB$ Token | TokenAmount$ VoteNumNumbers | TokenScript$ w_1_1_soldier | TokenOwner$ You | LegacyImage$ w 1 1 soldier cn2
```

Reply:
```suggestion
SVar:TrigVote:DB$ Vote | Defined$ Player | StoreVoteNum$ True | VoteType$ Evidence,Bribery | VoteEvidence$ DBEvidence | VoteBribery$ DBBribery | EachVote$ True
SVar:DBEvidence:DB$ Investigate | Num$ VoteNumEvidence
SVar:DBBribery:DB$ Token | TokenAmount$ VoteNumBribery | TokenScript$ c_a_treasure_sac | TokenOwner$ You

```

So I've tried this and it just doesn't make any tokens and for the life of me I cannot see why.

Reply:
You need to follow the pattern more closely

all that can be removed, instead work with SubAbility now:
`VoteEvidence$ DBEvidence | VoteBribery$ DBBribery | EachVote$ True`

Reply:
```SVar:TrigVote:DB$ Vote | Defined$ Player | StoreVoteNum$ True | VoteType$ Evidence,Bribery | SubAbility$ DBEvidence
SVar:DBEvidence:DB$ Investigate | Defined$ Self | Num$ VoteNumEvidence | SubAbility$ DBBribery
SVar:DBBribery:DB$ Token | TokenAmount$ VoteNumBribery | TokenScript$ c_a_treasure_sac | TokenOwner$ You```

Thank you for the tip, tool4ever! Got it working for the treasures but investigate makes no Clues.

Reply:
Thank you for the tip, tool4ever! Got it working for the treasures with the above but investigate makes no Clues.

Reply:
Investigate needs `Defined$ You`